fetch-curr-commit-id

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+# See https://stackoverflow.com/questions/14989858/get-the-current-git-hash-in-a-python-script
+
+import git
+
+repo = git.Repo(search_parent_directories=True)
+hexsha = repo.head.object.hexsha
+short_sha = repo.git.rev_parse(hexsha, short=True)
+print(f"{short_sha}")
```

Comment:
Didn't even think of this before, but isn't it a bit odd to use a template string formatter with a single string parameter? Ah, well. `print(short_sha)`???

Reply:
Fixed.